app/views/course/discussion/topics/_topics.html.slim

```diff
@@ -0,0 +1,4 @@
+- @topics.each do |topic|
+  = render partial: topic.actable_type.demodulize.underscore, locals: { topic: topic }
```

Comment:
If there are common things that should be shared, it should be in the views/layouts directory.

I don't think the different types of posts should share the partial, however. That seems like a leaky abstraction.